service/ocean/providers/azure/cluster.go

```diff
@@ -230,6 +230,72 @@ type ManagedServiceIdentity struct {
 	nullFields      []string
 }
 
+type RollSpec struct {
+	ID                        *string  `json:"id,omitempty"`
+	ClusterID                 *string  `json:"clusterId,omitempty"`
+	Comment                   *string  `json:"comment,omitempty"`
+	Status                    *string  `json:"status,omitempty"`
+	BatchSizePercentage       *int     `json:"batchSizePercentage,omitempty"`
+	BatchMinHealthyPercentage *int     `json:"batchMinHealthyPercentage,omitempty"`
+	RespectPDB                *bool    `json:"respectPdb,omitempty"`
+	NodePoolNames             []string `json:"nodePoolNames,omitempty"`
+	VngIds                    []string `json:"vngIds,omitempty"`
+	RespectRestrictScaleDown  *bool    `json:"respectRestrictScaleDown,omitempty"`
+	NodeNames                 []string `json:"nodeNames,omitempty"`
+
+	forceSendFields []string
+	nullFields      []string
+}
+
+type RollStatus struct {
+	ID                        *string    `json:"id,omitempty"`
+	ClusterID                 *string    `json:"oceanId,omitempty"`
+	Scope                     *string    `json:"scope,omitempty"`
+	Comment                   *string    `json:"comment,omitempty"`
+	Status                    *string    `json:"status,omitempty"`
+	Progress                  *Progress  `json:"progress,omitempty"`
+	RespectPDB                *bool      `json:"respectPdb,omitempty"`
+	RespectRestrictScaleDown  *bool      `json:"respectRestrictScaleDown,omitempty"`
+	BatchMinHealthyPercentage *int       `json:"batchMinHealthyPercentage,omitempty"`
+	CurrentBatch              *int       `json:"currentBatch,omitempty"`
+	NumOfBatches              *int       `json:"numOfBatches,omitempty"`
+	LaunchSpecIDs             []string   `json:"launchSpecIds,omitempty"`
```

Comment:
please re-check with actual response

Reply:
Removed LaunchSpecIDs and InstanceIDs attributes from structs as those were mistakenly added.